naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftCore.java

```diff
@@ -458,22 +472,29 @@ private void sendVote() {
             for (final String server : peers.allServersWithoutMySelf()) {
                 final String url = buildUrl(server, API_VOTE);
                 try {
-                    HttpClient.asyncHttpPost(url, null, params, new AsyncCompletionHandler<Integer>() {
+                    asyncRestTemplate.postForm(url, Header.EMPTY, params, String.class, new Callback<String>() {
                         @Override
-                        public Integer onCompleted(Response response) throws Exception {
-                            if (response.getStatusCode() != HttpURLConnection.HTTP_OK) {
-                                Loggers.RAFT
-                                        .error("NACOS-RAFT vote failed: {}, url: {}", response.getResponseBody(), url);
-                                return 1;
+                        public void onReceive(RestResult<String> result) {
+                            if (result.ok()) {
```

Comment:
Same suggest as above. Otherwise others code do not change in fact, but the author changed.